lib/widget/bidding.dart

```diff
@@ -15,157 +16,158 @@ import 'common.dart';
       ),
-      new Text(
+      Text(
         bidAmount.toString(),
         style: bigNumberTextStyle,
       ),
       iconWidget(
         context,
         Icons.arrow_forward,
-        () => store.dispatch(new IncrementBidAmountAction(upperBound)),
+        () => store.dispatch(IncrementBidAmountAction(upperBound)),
         bidAmount < upperBound,
       ),
     ],
   );
 }
 
 Widget submitButton(BuildContext context, Store<GameModel> store, bool loading, int bidAmount) =>
-    new RaisedButton(
+    RaisedButton(
         child: Text(AppLocalizations.of(context).submitBid, style: buttonTextStyle),
         onPressed: loading
             ? null
-            : () => store.dispatch(new SubmitBidAction(getSelf(store.state).id, bidAmount)));
+            : () => store.dispatch(SubmitBidAction(getSelf(store.state).id, bidAmount)));
 
 Widget cancelButton(BuildContext context, Store<GameModel> store, bool loading, Bid bid) =>
-    new RaisedButton(
-        color: Theme.of(context).accentColor,
-        child: new Text(AppLocalizations.of(context).cancelBid, style: buttonTextStyle),
-        onPressed: loading || bid == null ? null : () => store.dispatch(new CancelBidAction()));
-
-Widget bidding(Store<GameModel> store) {
-  return StoreConnector<GameModel, BiddingViewModel>(
-      converter: (store) => new BiddingViewModel._(
-            currentBalance(store.state),
-            getBidAmount(store.state),
-            requestInProcess(store.state, Request.Bidding),
-            myCurrentBid(store.state),
-            bidderNames(store.state),
-            haveGuessedBrenda(store.state),
-          ),
-      distinct: true,
-      builder: (context, viewModel) {
-        Haunt heist = currentHaunt(store.state);
-        bool auction = isAuction(store.state);
-
-        List<Widget> children = auction
-            ? [
-                new Container(
-                  padding: paddingTitle,
-                  child: new Text(AppLocalizations.of(context).auctionTitle.toUpperCase(),
-                      style: titleTextStyle),
-                ),
-                new Text(
-                  AppLocalizations.of(context).auctionDescription(heist.numPlayers),
-                  style: infoTextStyle,
-                ),
-              ]
-            : [
-                new Container(
-                  padding: paddingTitle,
-                  child: new Text(
-                    AppLocalizations.of(context).bidding,
-                    style: titleTextStyle,
-                  ),
-                ),
-              ];
-
-        String proposedBidText = viewModel.bid == null
-            ? AppLocalizations.of(context).noBid
-            : viewModel.bid.amount.toString();
-        children.addAll([
-          new Container(
-            padding: paddingSmall,
-            child: iconText(
-              new Icon(
-                Icons.attach_money,
-                size: 32.0,
+    RaisedButton(
+        color: HeistColors.peach,
```

Comment:
It'd be nice if all the colours came from the theme but don't worry now if it's not straightforward